<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Number Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QListWidget" name="listWidget"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QComboBox" name="comboBox">
           <item>
            <property name="text">
             <string>Sum</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Multiply</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Average</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="addButton">
           <property name="text">
            <string>Add...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="editButton">
           <property name="text">
            <string>Edit...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deleteButton">
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="spFunctionButton">
        <property name="text">
         <string>Special function...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearButton">
        <property name="text">
         <string>Clear...</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>98</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="ExitButton">
        <property name="text">
         <string>Exit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menunumber">
    <property name="title">
     <string>number</string>
    </property>
    <addaction name="actionadd"/>
    <addaction name="actionedit"/>
    <addaction name="actiondelet"/>
    <addaction name="actionSpecial_Function"/>
   </widget>
   <widget class="QMenu" name="menuNumber">
    <property name="title">
     <string>List</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuNumber"/>
   <addaction name="menunumber"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionadd"/>
   <addaction name="actionedit"/>
   <addaction name="actiondelet"/>
   <addaction name="separator"/>
   <addaction name="actionSpecial_Function"/>
  </widget>
  <action name="actionadd">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/build/add.png</normaloff>:/build/add.png</iconset>
   </property>
   <property name="text">
    <string>Add Number</string>
   </property>
  </action>
  <action name="actionedit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/build/pencil.png</normaloff>:/build/pencil.png</iconset>
   </property>
   <property name="text">
    <string>Edit Number</string>
   </property>
  </action>
  <action name="actiondelet">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/build/bin.png</normaloff>:/build/bin.png</iconset>
   </property>
   <property name="text">
    <string>Delete Number</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSpecial_Function">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/gears.png</normaloff>:/gears.png</iconset>
   </property>
   <property name="text">
    <string>Special Function</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ExitButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>752</x>
     <y>556</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addButton</sender>
   <signal>clicked()</signal>
   <receiver>actionadd</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>382</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editButton</sender>
   <signal>clicked()</signal>
   <receiver>actionedit</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>483</x>
     <y>382</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deleteButton</sender>
   <signal>clicked()</signal>
   <receiver>actiondelet</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>616</x>
     <y>382</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spFunctionButton</sender>
   <signal>clicked()</signal>
   <receiver>actionSpecial_Function</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>424</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
